src/python_bluesky_taskgraph/core/task.py

```diff
@@ -48,15 +51,14 @@ class BlueskyTask(Generic[InputType]):
 
     def __init__(self, name: str):
         self._name: str = name
-        # TODO: Do we want logging at the Task, DecisionEngine or ControlObject level?
-        self._logger = logging.getLogger("BlueskyTask")
+        self._logger = logging.getLogger(self.name)
```

Comment:
Ditto for logger

Reply:
Tasks definitely should log on the level of individual instances of the task. Have changed to extend the module level logger, so logs can be bundled or split apart appropriately.